src/bridge/SequencerInbox.sol

```diff
@@ -374,26 +374,32 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
         IGasRefunder gasRefunder,
         uint256 prevMessageCount,
         uint256 newMessageCount,
-        bytes memory quote
+        bytes memory batcherSignatureAndHotshotHeight
     ) external refundsGas(gasRefunder, IReader4844(address(0))) {
         // solhint-disable-next-line avoid-tx-origin
         if (msg.sender != tx.origin) revert NotOrigin();
         if (!isBatchPoster[msg.sender]) revert NotBatchPoster();
 
         // take keccak2256 hash of all the function arguments except the quote
+        (uint256 hotshotHeight, bytes memory signature) = abi.decode(
+            batcherSignatureAndHotshotHeight,
+            (uint256, bytes)
+        );
+
         bytes32 reportDataHash = keccak256(
             abi.encode(
                 sequenceNumber,
                 data,
                 afterDelayedMessagesRead,
                 address(gasRefunder),
                 prevMessageCount,
-                newMessageCount
+                newMessageCount,
+                hotshotHeight
             )
         );
         // verify the quote for the batch poster running in the TEE
-        espressoTEEVerifier.verify(quote, reportDataHash);
-        emit TEEAttestationQuoteVerified(sequenceNumber);
+        espressoTEEVerifier.verify(signature, reportDataHash);
+        emit LastestHotshotHeight(hotshotHeight);
```

Comment:
Shouldn't we still emit a log about the signature being verified at the sequenc number?

Reply:
true, we can add sequencerNumber as well!

Reply:
https://github.com/EspressoSystems/nitro-contracts/pull/53/commits/686f7ae66599fa385320854c9344934676ee970b